### PR TITLE
Stop getting messages on 401/404

### PR DESCRIPTION
**Pull request by gary-kim**
Signed-off-by: Gary Kim <gary@garykim.dev>

### CODE REVIEW

File: room/room.go [hunk 1]
```diff
@@ -127,6 +129,19 @@ func (t *TalkRoom) ReceiveMessages(ctx context.Context) (chan ocs.TalkRoomMessag
...
+			if res.StatusCode == 404 {
+				_ = res.Body.Close()
+				c <- ocs.TalkRoomMessageData{Error: ErrRoomNotFound}
+				return
+			}
+			if res.StatusCode == 401 {
+				_ = res.Body.Close()
+				c <- ocs.TalkRoomMessageData{Error: ErrUnauthorized}
+				return
+			}
```
Comment (nickvergessen) [inline room/room.go:143]:
```suggestion
			}
			if res.StatusCode == 429 {
				_ = res.Body.Close()
				c <- ocs.TalkRoomMessageData{Error: ErrTooManyRequests}
				return
			}
```

Comment (gary-kim) [reply room/room.go:143]: Let's go with this for now and open an issue for maybe having a config for wait on 429?

Comment (nickvergessen) [reply room/room.go:145]: Well 429 happens when you try to brute force login credentials or your token expired and you therefor use invalid credentials

Comment (gary-kim) [reply room/room.go:145]: My bad. In that case, let's just quit.

**Pull request merged**